changelog/2.075.0_pre.dd

```diff
@@ -2,7 +2,7 @@ Ddoc
 
 $(CHANGELOG_NAV_LAST 2.074.1)
 
-$(VERSION Jul 16, 2017, =================================================,
+$(BETA_VERSION Jul 16, 2017, =================================================,
```

Comment:
To avoid doing this manually, there will be `--release-type=beta` (https://github.com/dlang/tools/pull/251)